main.go

```diff
@@ -152,12 +150,15 @@ func main() {
 		router.Handle("/cookies{uri:.*}", createReverseProxy("cookies", cookiesControllerURL))
 	}
 
-	if cfg.DatasetRoutesEnabled {
-		router.Handle("/datasets/{uri:.*}", createReverseProxy("datasets", datasetControllerURL))
-		router.Handle("/feedback{uri:.*}", createReverseProxy("feedback", datasetControllerURL))
-		router.Handle("/filters/{uri:.*}", createReverseProxy("filters", filterDatasetControllerURL))
-		router.Handle("/filter-outputs/{uri:.*}", createReverseProxy("filter-output", filterDatasetControllerURL))
+	router.Handle("/datasets/{uri:.*}", createReverseProxy("datasets", datasetControllerURL))
+	router.Handle("/filters/{uri:.*}", createReverseProxy("filters", filterDatasetControllerURL))
+	router.Handle("/filter-outputs/{uri:.*}", createReverseProxy("filter-output", filterDatasetControllerURL))
+	router.Handle("/feedback{uri:.*}", createReverseProxy("feedback", datasetControllerURL))
```

Comment:
On testing in develop, it seems that we can't create the same route twice, and the first handler for that route will always get used - the feedbackEnabled flag setting feedbackControllerURL is at this point redundant.

I'd suggest moving L156 to and else statement off if cfg.FeedbackEnabled, or refactoring them both so the if statement is setting a path, and then the route is set once, e.g.

```suggestion

        feedbackURL := datasetControllerURL
        if cfg.FeedbackEnabled {
                feedbackURL = feedbackControllerURL
        }
	router.Handle("/feedback{uri:.*}", createReverseProxy("feedback", feedbackURL))
```